Apps/Examples/Examples/All Examples/Localization/LocalizationSettingsExample.swift

```diff
@@ -0,0 +1,175 @@
+import Foundation
+import MapboxMaps
+
+/// An example to show how to set language and worldview settings with new settings API.
+/// When set, the tiles will be loaded in localized language/worldview if the query
+/// parameters are supported.
+///
+/// In order to get localized tiles (the server side localization), create a Setting service instance using SettingsServiceFactory,
+/// and set `MapboxCommonSettings.language` or `MapboxCommonSettings.worldview`.
+///
+/// `SettingsServiceInterface` stores the provided language/worldview as a key:value pair in `UserDefaults`
+/// and try to apply it to the tiles. If the values are supported, the localized tiles will be loaded onto the map,
+/// otherwise `MapEvents.EventKind.mapLoadingError` event will be emitted.
+///
+/// The language parameter should be one of the BCP-47 tags and the worldview parameter one of the ISO 3166-1 alpha-2 codes.
```

Comment:
> should be one of the BCP-47 tags

Can we make an example here? I don't expect many developers known what is the BCP-47 is 🙂 

> one of the ISO 3166-1 alpha-2 codes

Same here